README.md

```diff
@@ -1 +1,36 @@
-Rails app generated with [lewagon/rails-templates](https://github.com/lewagon/rails-templates), created by the [Le Wagon coding bootcamp](https://www.lewagon.com) team.
+<h2> 🎓 Mojo Learning Labs </h2>
+
+Mojo Learning Labs is an assignment setting and marking tool that allows teachers to conduct formative assessments in an easy, engaging manner. Through this web app, the teacher can create a class, send email invites to students, set them assignments and obtain an overview of class performance. Likewise, students are able to join classes, answer assignments and receive instant feedback.
+
+
+<h2> Built with: </h2>
+<p>
+  
+- Figma (wireframing/prototyping)
+- Ruby on Rails
+- git
+- GitHub
+- Heroku
+- PostgreSQL
+- HTML
+- CSS
+- Bootstrap
+  
+</p>  
+
+<h2> Product screenshots </h2>
+
+<h4> Teacher's dashboard: </h4>
+
+<p float="left">
+  
+<img src="https://user-images.githubusercontent.com/82147496/162393477-f7cb24e4-086b-4822-9ba6-6a03b8592f41.png" width="600">
+<img src="https://user-images.githubusercontent.com/82147496/162395362-1d038804-5a4b-4e86-9820-546ed4372c6c.png" width="600">
+  
+</p>
+
+<h4> Student's dashboard: </h4>
+
+
+<img src="https://user-images.githubusercontent.com/82147496/162397603-751edc60-eacb-4e38-9b84-3655be6e77c6.png" width="600">
```

Comment:
A lot of images have capitals in the assets/images folder. recommend to only use lower case since it can me confusing when switching between capital and in capital sensitive operating systems. I.e. it might work on windows but not on linux. since its case sensitive.